doc/getting_started/index.rst

```diff
@@ -392,14 +392,20 @@ Install build tools and dependencies
 ARCN development is supported on popular Linux distributions,
 each with their own way to install development tools:
 
+  .. notes::
```

Comment:
directive name is misspelled and we can simplify the wording of the note:

    .. note::
       ACRN uses ``menuconfig``, a python3 text-based user interface (TUI) for configuring
       hypervisor options and using python's ``kconfiglib`` library.

Reply:
Will update. Thanks!